AppCUI/include/AppCUI.hpp

```diff
@@ -1370,41 +1458,41 @@ namespace Utils
         {
         }
     };
-    template <uint16 Size>
-    class FixSizeString
+    template <uint16 Size, typename T, typename T_view>
+    class GenericFixSizeString
     {
         static_assert(Size > 0);
-        char data[Size + 1];
+        T data[Size + 1];
```

Comment:
poate un static assert in clasa asta ca T e POD?

Reply:
fixed